src/libs/README.md

```diff
@@ -82,3 +82,15 @@ data structures.
     libelektra-invoke.so
 
 **[libinvoke](invoke/)** provides a simple API allowing us to call functions exported by plugins.
+
+### IO
+
+    libelektra-io.so
+
+**[io](io/)** provides the common [I/O binding API](https://doc.libelektra.org/api/current/html/group__kdbio.html).
```

Comment:
a very short description would be nice. It is unclear what "I/O" is in this context.